code.mdx

```diff
@@ -76,7 +77,8 @@ class HelloWorld {
 ````
 
 ### Title
-Add a title after the programming language to label your code example. Titles can be any string on a single line.
+
```

Comment:
I think we can add the `You can specify a title...` after the first example using the default example codeblock. Then frame it like: you can also....